gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -238,6 +238,12 @@ message TripUpdate {
       // NOTE: This field is still experimental, and subject to change. It may be
       // formally adopted in the future.
       UNSCHEDULED = 3;
+
+        // The vehicle's relationship with the static schedule differs in a way that impacts the rider experience,
+        // such as a vehicle not picking up passengers because a vehicle is "drop-off only" to catch up back to
+        // schedule when it's running late. StopTimeProperties must included to specify how the relationship
+        // has been modified.
+      MODIFIED = 4; 
```

Comment:
The inclusion of the `MODIFIED` enum deviates a bit from our original plan with [GTFS-ServiceChanges v3.1](https://docs.google.com/document/d/1oPfQ6Xvui0g3xiy1pNiyu5cWFLhvrTVyvYsMVKGyGWM/edit#heading=h.mm6bblup6h8d). 

We intentionally did not want to introduce any enums that could be interpreted as being able to alter the stop sequence of a trip (beyond the allowances for `assigned_stop_id`). This was the justification behind keeping `SCHEDULED` as the schedule relationship, because the stop sequence still matches the schedule data. It seemed sufficient at the time that the presence of `StopTimeProperties` was sufficient to indicate that those properties had changed from the static data, and a new enum wasn't needed. I'm open to counter-arguments for this, though, if others feel it's needed.

If we keep this `MODIFIED` enum, then it seems it should be used for any data within `StopTimeProperties` (including the already existing `assigned_stop_id`). It should also specifically stated that it does NOT apply to any of the fields outside of `StopTimeProperties` - in other words, you can't just say `MODIFIED` and throw any combination of `stop_id` and `stop_sequence` in a `StopTimeUpdate` and expect a consumer to change the trip this way.

A few other questions:
1. How should I indicate that I don't have arrival predictions for a stop that I `MODIFIED` (i.e., if we can't use `NO_DATA`)?
2. Can we used `MODIFIED` and `StopTimeProperties` for exact_times=0 trips (i.e., if we can't use `UNSCHEDULED`)?